### PR TITLE
feat(berlin_summit_23): new blogpost [#98]

### LINKED ISSUE #98: AWS Berlin Summit 2023
Title: AWS Berlin Summit '23 Roundup

Highlights, tips, recommendations from this years summit.

### PR DESCRIPTION
**Pull request by flyck**
Closes #98 

Open Issues:
- [x] Let the blog post sit for a day
- [x] Optimize the used markdown images (experimental feature https://docs.astro.build/en/guides/images/)
- [x] Squooshify the images

### CODE REVIEW

**Comment by hendriknielaender** [comment]
https://docs.astro.build/en/guides/images/#srcassets-experimental

we are already using it:
Bildschirmfoto 2023-05-08 um 20 16 32

File: data/blog/berlin_summit_2023_review.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    ![](/src/assets/berlin_summit_2023_review/mad202_ops_meetings.jpg)
+  - Change happens by treating service teams as independent startups. They are empowered to make
+    their own decisions. They can decide, which feature they believe will serve the customer
+    best. As a trade-off for more autonomy, they get held accountable only through KPIs, looking
+    at the end results.
+  - Following Werner Vogels approach of [working
+    backwards](https://www.allthingsdistributed.com/2006/11/working_backwards.html) from customer
+    needs is one of the hardest and most important things to put into practice.
+- MAD303 (Unlocking Busines Agility with Event-Driven Architectures), a great architecture talk,
+  really hammered home the point of puttung queues behind event receivers. Allowing any subscriber
```
Comment (hendriknielaender) [inline data/blog/berlin_summit_2023_review.md:86]:
```suggestion
  really hammered home the point of putting queues behind event receivers. Allowing any subscribers
```

File: data/blog/berlin_summit_2023_review.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+# First Impressions
+
+Heading straight into the "AWS World" area, waiting there was a difficult and fast-paced AWS quiz,
+along with AWS socks and stickers. The quiz was exceptional, and I really wish they would bring it
+out as an app for training.
+
+A cloud style escape room, a Four In A Row (Vier gewinnt) against AI, and AWS Principal Cloud
+Architects already waiting on the sidelines, made me eager to come back later and explore
+more. The Four In A Row was interesting, as the metrics were published through Athena into a
```
[the remark this answers is not in the document]
Comment (flyck) [reply data/blog/berlin_summit_2023_review.md:25]: done ✅

File: data/blog/berlin_summit_2023_review.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+- Data Visibility in the company is key.
+- Consider the impact a customized experience could have on your customers and revenue.
+- When building an AI, look at existing use-cases over here https://aiexplorer.aws.amazon.com/
+
+Highlights and Learnings:
+- Quite possibly my favorite talk was ANT301 (Frictionless Compliant Data & Analytics Environments
+  at Merck). Kudos to [John Mousa](https://www.linkedin.com/in/johnmousa/) for making this
+  presentaiton exceptionally kind, enjoyable and insightful. Truly a talk of its own league.
+  ![](/src/assets/berlin_summit_2023_review/ant301_people.jpg)
+  - You make change happen by identifying innovators and empowering them
```
[the remark this answers is not in the document]
Comment (flyck) [reply data/blog/berlin_summit_2023_review.md:68]: i'd say this doesnt need to be uppercase

File: data/blog/berlin_summit_2023_review.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+While I thought the keynote was underwhelming due to its overly heroic focus on the situation in
+Ukraine, and the boring talk from Siemens, what followed was much better. After past years of swag
+hunting, this year I was satisfied with the AWS socks and stickers I got early on, and got sucked
+into one talk after the other.
+
+# Ideas and Highlights
+
+Opposite to the Nodeconf we visited last year, the Berlin Summit didn't really have a common
+theme - well, except for the usage of AWS. The different stories and learnings provided by big AWS
+customers, as well as from AWS themselves, still provided a constant stream of brainfood and new
```
[the remark this answers is not in the document]
Comment (flyck) [reply data/blog/berlin_summit_2023_review.md:55]: https://www.dict.cc/?s=brainfood correct in this case

File: data/blog/berlin_summit_2023_review.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+compressed into one, which made it feel that much more exciting. Polished booths, a diverse crowd,
+plenty of snacks, exciting challenges and good talks.
+
+During this Summit Hendrik had stayed away, in part because of mixed reviews from
+previous years, so only I went. And it was worth it!
+
+# First Impressions
+
+Heading straight into the "AWS World" area, waiting there was a difficult and fast-paced AWS quiz,
+along with AWS socks and stickers. The quiz was exceptional, and I really wish they would bring it
```
[the remark this answers is not in the document]
Comment (flyck) [reply data/blog/berlin_summit_2023_review.md:20]: i guess it's a nominalized adjective here

File: data/blog/berlin_summit_2023_review.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+---
+
+This years Berlin Summit showed why it needs to be taken seriously. The usual two day format got
+compressed into one, which made it feel that much more exciting. Polished booths, a diverse crowd,
+plenty of snacks, exciting challenges and good talks.
+
+During this Summit Hendrik had stayed away, in part because of mixed reviews from
+previous years, so only I went. And it was worth it!
+
+# First Impressions
```
[the remark this answers is not in the document]
Comment (flyck) [reply data/blog/berlin_summit_2023_review.md:17]: https://www.grammarly.com/blog/capitalization-in-the-titles/

Titles and headings are capitalized differently

File: data/blog/berlin_summit_2023_review.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+imageCreditUrl: https://double-trouble.dev
+tags: [conf, aws]
+---
+
+This years Berlin Summit showed why it needs to be taken seriously. The usual two day format got
+compressed into one, which made it feel that much more exciting. Polished booths, a diverse crowd,
+plenty of snacks, exciting challenges and good talks.
+
+During this Summit Hendrik had stayed away, in part because of mixed reviews from
+previous years, so only I went. And it was worth it!
```
[the remark this answers is not in the document]
Comment (flyck) [reply data/blog/berlin_summit_2023_review.md:15]: years seems correct
image

File: data/blog/berlin_summit_2023_review.md [hunk 8]
```diff
@@ -0,0 +1,134 @@
+---
+publishDate: "May 8 2023"
+title: "Berlin Summit 2023 Writeup"
```
[the remark this answers is not in the document]
Comment (flyck) [reply data/blog/berlin_summit_2023_review.md:3]: Is indeed more catchy 👍🏻

File: data/blog/berlin_summit_2023_review.md [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+image: ~/assets/images/thumbnails/berlin_summit_keynote.jpg
+imageCreditUrl: https://double-trouble.dev
+tags: [conf, aws]
+---
+
+This years Berlin Summit showed why it needs to be taken seriously. The usual two day format got
+compressed into one, which made it feel that much more exciting. Polished booths, a diverse crowd,
+plenty of snacks, exciting challenges and good talks.
+
+During this Summit Hendrik had stayed away, in part because of mixed reviews from
```
[the remark this answers is not in the document]
Comment (flyck) [reply data/blog/berlin_summit_2023_review.md:14]: Maybe lets do some variation of this sentence, so its clear why it was written in the I-form, while the blog has 2 authors. 🤔 

But we could also remove it.

File: data/blog/berlin_summit_2023_review.md [hunk 10]
```diff
@@ -0,0 +1,134 @@
+---
+publishDate: "May 8 2023"
+title: "Berlin Summit 2023 Writeup"
+description: "Highlights, tips, recommendations from this years summit."
```
[the remark this answers is not in the document]
Comment (flyck) [reply data/blog/berlin_summit_2023_review.md:4]: Pretty cool for SEO indeed and sounds great 👍🏻 

But on the downside it blows up the preview 🤔 then i would prefer something short
image

**Comment by flyck** [comment]
> https://docs.astro.build/en/guides/images/#srcassets-experimental
> 
> we are already using it: Bildschirmfoto 2023-05-08 um 20 16 32

ok then i guess only the preview is broken.. should be working according to the docs 👍🏻 
https://docs.astro.build/en/guides/assets/#update-your-markdown-mdx-and-markdoc-files

File: data/blog/berlin_summit_2023_review.md [hunk 11]
```diff
@@ -0,0 +1,134 @@
...
+plenty of snacks, exciting challenges and good talks.
+
+During this Summit Hendrik had stayed away, in part because of mixed reviews from
+previous years, so only I went. And it was worth it!
+
+# First Impressions
+
+Heading straight into the "AWS World" area, waiting there was a difficult and fast-paced AWS quiz,
+along with AWS socks and stickers. The quiz was exceptional, and I really wish they would bring it
+out as an app for training.
```
[the remark this answers is not in the document]
Comment (flyck) [reply data/blog/berlin_summit_2023_review.md:21]: yeah they had a real life quiz there where you drag and drop answers into true or false buckets. the questions were falling inside little clouds from the sky and you had to react fast. would make for a crazy app to learn more about aws 👀

**Comment by flyck** [comment]
Sorry for not running a spellchecker first 💩 finally did it

File: data/blog/berlin_summit_2023_review.md [hunk 12]
```diff
@@ -0,0 +1,134 @@
...
+publishDate: "May 8 2023"
+title: "Berlin Summit 2023 Writeup"
+description: "Highlights, tips, recommendations from this years summit."
+image: ~/assets/images/thumbnails/berlin_summit_keynote.jpg
+imageCreditUrl: https://double-trouble.dev
+tags: [conf, aws]
+---
+
+This years Berlin Summit showed why it needs to be taken seriously. The usual two day format got
+compressed into one, which made it feel that much more exciting. Polished booths, a diverse crowd,
```
[the remark this answers is not in the document]
Comment (flyck) [reply data/blog/berlin_summit_2023_review.md:11]: using 'that' here in this way is saying that the more compressed it is, the more exciting it was.. its like a special expression. I guess I'd keep it to make the sentence more interesting 🤔

**Comment by hendriknielaender** [comment]
Ron Nod

**Pull request merged**